### PR TITLE
firewalld vars

### PR DESCRIPTION
**Pull request by drmatthews**
Changes:

- Role name prefix for `mirsg.infrastructure.firewalld` variables
- Move firewalld role variables out of `install_monitoring.yml` playbook

### CODE REVIEW

File: .github/workflows/molecule-monitoring.yml [hunk 1]
```diff
@@ -43,7 +43,6 @@ jobs:
           molecule-plugins[docker] \
           docker \
           passlib \
-          bcrypt==4.0.1 \
```
Comment (drmatthews) [inline .github/workflows/molecule-monitoring.yml:46]: Using `bcrypt` with the `password_hash` filter was causing an exception.

File: playbooks/install_monitoring.yml [hunk 2]
```diff
@@ -29,12 +29,3 @@
   roles:
     - role: mirsg.infrastructure.monitoring_client
     - role: mirsg.infrastructure.firewalld
-      vars:
```
Comment (drmatthews) [inline playbooks/install_monitoring.yml:32]: Am now setting these at the inventory level instead.

File: roles/monitoring_client/templates/node_exporter_web.yml.j2 [hunk 3]
```diff
@@ -2,4 +2,4 @@ tls_server_config:
   cert_file: node_exporter.crt
   key_file: node_exporter.key
 basic_auth_users:
-  {{ monitoring_client_exporter_username }}: {{ monitoring_client_exporter_password | password_hash('bcrypt') }}
```
Comment (drmatthews) [inline roles/monitoring_client/templates/node_exporter_web.yml.j2:5]: Using `bcrypt` with `passlib` was causing an exception when using the `password_hash` filter.

File: playbooks/install_monitoring.yml (hunk 2 above)
Comment (p-j-smith) [reply playbooks/install_monitoring.yml:32]: nice, makes sense to put them in the inventory

**Review by p-j-smith — APPROVED** [review]
looks good

**Pull request merged**